database/migrations/2023_06_29_151653_add_photo_source.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('photos', function (Blueprint $table) {
+            $table->string('source')->nullable();
```

Comment:
Nit-pick, but it's nice to keep timestamp columns (created_at, updated_at) last

https://laravel.com/docs/10.x/migrations#column-modifiers

```suggestion
            $table->string('source')->after('description')->nullable();
```